components/EventForm/index.js

```diff
@@ -0,0 +1,50 @@
+import Link from "next/link";
+import { useState } from "react";
+import styled from "styled-components";
+import { useRouter } from "next/router";
+
+const Paragraph = styled.p`
+  font-size: small;
+  color: gray;
+`;
+
+export default function EventForm({ onAddEvent }) {
+  const router = useRouter();
+  function handleSubmit(event) {
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    const data = Object.fromEntries(formData);
+
+    onAddEvent(data);
+    alert("🎉You have added your Event successfully!");
+    router.push("/my-events");
+    event.target.reset();
```

Comment:
If you redirect the user to another page after submitting the form, you do not need to reset the form. :)
```suggestion
    router.push("/my-events");

```